go/chat/teams.go

```diff
@@ -414,7 +432,7 @@ func (t *ImplicitTeamsNameInfoSource) makeNameInfo(ctx context.Context, team *te
 		return res, errors.New("blank TLF ID given")
 	}
 	res.CanonicalName = impTeamName.String()
-	if res.CryptKeys, err = getTeamKeys(ctx, team, public); err != nil {
+	if res.CryptKeys, err = getTeamCryptKeys(ctx, team, public); err != nil {
```

Comment:
Is this still called all the time? Does NameInfo need keys on it? If refactoring chat is a pain we can cache the verification of keys on teams so getting all keys is fastish again.

Reply:
I'm not quite done, I am going to refactor a little more and make this call very rare.